qcfractal/tests/test_portal.py

```diff
@@ -44,15 +44,15 @@ def test_options_portal(test_server):
 
 def test_collection_portal(test_server):
 
-    db = {"collection": "TorsionDrive", "name": "Torsion123", "something": "else", "array": ["54321"]}
+    db = {"collection": "torsiondrive", "name": "Torsion123", "something": "else", "array": ["54321"]}
```

Comment:
This brings up an interesting point. Should we make db calls case-insensitive for things like the collection name? We would have to store a map of either all Lcase or Ucase names to the class objects, but I can see this being an issue.

Reply:
Yea, have been toying around with this. Currently we wrap it correctly on the client side of things, but not on the mongo side. Same issue with `name` where we might want `display_name` or similar where the case is kept.

Reply:
I think this is fine fore now, but something we can think about for future releases.